src/explorer/buildspecs.jl

```diff
@@ -0,0 +1,197 @@
+## Mimi UI
+using Mimi
+using DataFrames
+using JSON 
+
+function getspeclist(model::Mimi.Model)
```

Comment:
No need to write ``Mimi.Model``, ``Model`` should be fine (because this function is defined in the same module).